recipe/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-sh Configure -de -Dprefix=$PREFIX -Duserelocatableinc
+sh Configure -Dusethreads -Duserelocatableinc -Dprefix=$PREFIX -de
```

Comment:
So I believe there was something special about how these needed to be ordered. See issue ( https://github.com/conda-forge/perl-feedstock/issues/4 ). If you could explain a bit why this order is better, that would be very helpful.